standalone/pruntime/enclave/src/system/gk.rs

```diff
@@ -86,11 +87,13 @@ impl WorkerInfo {
 //
 // For simplicity, we also ensure that each gatekeeper responds (if registered on chain)
 // to received messages in the same way.
-pub(super) struct Gatekeeper<MsgChan> {
+pub(super) struct Gatekeeper {
```

Comment:
This will break the tests in gk.rs. Consider `impl MessageChannel for Sr25519MessageChannel`.
Run `cargo check --features tests` in the enclave to see if the tests compiles.